project/src/components/add-review-form/add-review-form.tsx

```diff
@@ -1,42 +1,54 @@
+import { useState } from 'react';
+
 type RatingInputProps = {
   name: string;
   index: number;
+  onChecked: (id: string) => void;
 };
 
-function RatingInput({ name, index }: RatingInputProps): JSX.Element {
+function RatingInput({ name, index, onChecked }: RatingInputProps): JSX.Element {
   return (
     <>
-      <input className="rating__input" id={`star-${index}`} type="radio" name="rating" value={index} />
+      <input
+        className="rating__input"
+        id={`star-${index}`}
+        type="radio"
+        name="rating"
+        value={index}
+        onChange={() => onChecked(`star-${index}`)}
```

Comment:
Так же не хватает checked свойства, лучше его указывать
```suggestion
        onChange={() => onChecked(`star-${index}`)}
        checked={index === currentRating(из родителя)}
```